aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,204 +1,26 @@
 ---
 remote_endpoint: ~
-name: "v1.10-plus"
+name: "v1.11"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverrideOld: # This is translated to `gas_schedule::set_gas_schedule(...)`.
+      - DefaultGasWithOverride:
```

Comment:
Release builder has been updated for randomness framework, and `DefaultGasWithOverride` will be translated to a function that only the new framework has.

But since this step is before the framework upgrade, the execution will fail.
I made `DefaultGasWithOverrideOld` entry for one-time use to translate to the old gas function. In the future we can come back to `DefaultGasWithOverride` as usual.

Reply:
> Release builder has been updated for randomness framework, and DefaultGasWithOverride will be translated to a function that only the new framework has.

@zjma , when was this updated? (cc: @vgao1996 )

Reply:
#12228

Reply:
and #12112